Solutions/GitHub/Parsers/GitHubDependabotData.txt

```diff
@@ -0,0 +1,179 @@
+//     "deployments_url": "https://api.github.com/repos/Codertocat/Hello-World/deployments",
+//     "created_at": "2019-05-15T15:19:25Z",
+//     "updated_at": "2019-05-15T15:19:27Z",
+//     "pushed_at": "2019-05-15T15:20:32Z",
+//     "git_url": "git://github.com/Codertocat/Hello-World.git",
+//     "ssh_url": "git@github.com:Codertocat/Hello-World.git",
+//     "clone_url": "https://github.com/Codertocat/Hello-World.git",
+//     "svn_url": "https://github.com/Codertocat/Hello-World",
+//     "homepage": null,
+//     "size": 0,
+//     "stargazers_count": 0,
+//     "watchers_count": 0,
+//     "language": null,
+//     "has_issues": true,
+//     "has_projects": true,
+//     "has_downloads": true,
+//     "has_wiki": true,
+//     "has_pages": true,
+//     "forks_count": 0,
+//     "mirror_url": null,
+//     "archived": false,
+//     "disabled": false,
+//     "open_issues_count": 2,
+//     "license": null,
+//     "forks": 0,
+//     "open_issues": 2,
+//     "watchers": 0,
+//     "default_branch": "master"
+//   },
+//   "sender": {
+//     "login": "github",
+//     "id": 9919,
+//     "node_id": "MDEyOk9yZ2FuaXphdGlvbjk5MTk=",
+//     "avatar_url": "https://avatars1.githubusercontent.com/u/9919?v=4",
+//     "gravatar_id": "",
+//     "url": "https://api.github.com/users/github",
+//     "html_url": "https://github.com/github",
+//     "followers_url": "https://api.github.com/users/github/followers",
+//     "following_url": "https://api.github.com/users/github/following{/other_user}",
+//     "gists_url": "https://api.github.com/users/github/gists{/gist_id}",
+//     "starred_url": "https://api.github.com/users/github/starred{/owner}{/repo}",
+//     "subscriptions_url": "https://api.github.com/users/github/subscriptions",
+//     "organizations_url": "https://api.github.com/users/github/orgs",
+//     "repos_url": "https://api.github.com/users/github/repos",
+//     "events_url": "https://api.github.com/users/github/events{/privacy}",
+//     "received_events_url": "https://api.github.com/users/github/received_events",
+//     "type": "Organization",
+//     "site_admin": false
+//   }
+
+githubscanaudit_CL 
+| where action_s in ('create', 'dismiss', 'resolve')
+| extend EventType='RepositoryVulnerabilityAlert'
+| extend alert = todynamic(alert_s),
+    organization = todynamic(organization_s),
+    repository = todynamic(repository_s),
+    sender = todynamic(sender_s) 
+| extend alertcreatedate = alert.created_at, alertaffectedrange = alert.affected_range,   
+    alertexternalidentifier= alert.external_identifier,    
+    alertghsaid = alert.ghsa_id,
+    alertseverity = alert.severity,
+    repositoryfullname = repository.full_name,
+    repositoryOwnerlogin = repository.owner.login,
+    repositoryurl = repository.url,       
+    senderlogin = sender.login,
+    sendertype = sender.type,
+    action=action_s
+| where isempty(alertexternalidentifier) == false 
```

Comment:
You can use the function isnotempty(alertexternalidentifier) here.
This is applicable to other Parsers as well.

Reply:
done